.github/workflows/test.yaml

```diff
@@ -9,7 +9,11 @@ on:
       working-directory:
         type: string
         description: The working directory for jobs
-        default: './'
+        default: "./"
```

Comment:
I'm curious about the change from single to double quotes, why is that?

Reply:
The file uses both single quotes and double quotes. I used a formatter (YAML extension by Red Hat on vscode) to enforce using double quotes. Double quotes is chosen as less change was required compared to using single qoutes.